frontend/app/mocks/subscription-api.server.ts

```diff
@@ -105,6 +107,20 @@ export function getSubscriptionApiMockHandlers() {
       });
     }),
 
+    //
+    // Handler for patch request to update user by userId
+    //
+    http.patch('https://api.cdcp.example.com/api/v1/users/:userId', async ({ params, request }) => {
+      log.debug('Handling request for [%s]', request.url);
```

Comment:
```suggestion
      log.debug('Handling PATCH request for [%s]', request.url);
```

Originally these log lines didn't need to distinguish between GET, POST, PATCH, etc.. but they kind of do now.